PyISY/Connection.py

```diff
@@ -154,10 +154,22 @@ def getNodes(self):
         result = self.request(req_url)
         return result
 
+    # Get the device notes, currently only the Spoken property is saved.
     def getNodeNotes(self,nid):
         req_url = self.compileURL(['nodes', nid, 'notes'])
         result = self.request(req_url)
-        return result
```

Comment:
I'd rather have the unparsed XML be returned from this method. No other methods in this class parse their XML results.